### PR TITLE
Add new translations (close #282)

### LINKED ISSUE #282: Keeping track of untranslated strings.
- [x] More buttons
- [x] Premium subscriber popup
- [x] Add a new package button
- [x] Probably the subscriptions menu (can't see it since I'm premium)
- [x] Navigation
- [x] Shared image

I'll add more every time I find one

### PR DESCRIPTION
**Pull request by Mr-KayJayDee**
Added new translation !

Waiting for #282 to be closed to mark ready

### CODE REVIEW

**Comment by florian-lefebvre** [comment]
Let's do the opposite, we'll close this issue with this PR

File: src/app/[locale]/onboarding/_components/Footer.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ function TrustDialog({
                     className="w-full"
                     onClick={() => setOpen(false)}
                   >
-                    Got it!
+                    {t("onboarding.routesLabels./intro/1")}
```
Comment (florian-lefebvre) [inline src/app/[locale]/onboarding/_components/Footer.tsx:87]: Let's add a dedicated key here

File: src/app/[locale]/onboarding/loading/_components/QueueDisplay.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ function QueueExplanationDialog({
                     className="w-full"
                     onClick={() => setOpen(false)}
                   >
-                    Got it!
+                    {t("onboarding.routesLabels./intro/1")}
```
Comment (florian-lefebvre) [inline src/app/[locale]/onboarding/loading/_components/QueueDisplay.tsx:75]: Let's add a dedicated key here

File: src/components/StaticShareImage.tsx [hunk 3]
```diff
@@ -19,6 +20,7 @@ export type Props = {
   };
   topDMS: ListElement[];
   topGuilds: ListElement[];
+  t: ReturnType<typeof useTranslation>["t"];
```
Comment (florian-lefebvre) [inline src/components/StaticShareImage.tsx:23]: Did you test it? All good if it works

File: src/app/[locale]/(app)/_components/SharePopup.tsx [hunk 4]
```diff
@@ -56,6 +56,7 @@ function useImageData() {
         count: formatNumber(guild.message_count),
       };
     }),
+    t: useTranslation().t,
```
Comment (florian-lefebvre) [inline src/app/[locale]/(app)/_components/SharePopup.tsx:59]: I'd prefer to have a `useTranslation` in `useGenerateImg` and pass `t` to `StaticShareImage` directly, not through `generate`

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]

File: src/components/StaticShareImage.tsx (hunk 3 above)
Comment (Mr-KayJayDee) [reply src/components/StaticShareImage.tsx:23]: Yes, this has been tested and works. The issue here was that I cannot import a hook inside a hook that is also inside another hook 😅. This was the solution I came with.

File: src/app/[locale]/(app)/_components/SharePopup.tsx (hunk 4 above)
Comment (Mr-KayJayDee) [reply src/app/[locale]/(app)/_components/SharePopup.tsx:59]: As for this one, yes I'll fix that !👍

File: src/components/StaticShareImage.tsx (hunk 3 above)
Comment (florian-lefebvre) [reply src/components/StaticShareImage.tsx:23]: Yes it's normal if you can't import it in this component I think

Comment (Mr-KayJayDee) [reply src/components/StaticShareImage.tsx:23]: Yes, this helps avoiding hook loops 😅

File: src/app/[locale]/(app)/_components/SharePopup.tsx (hunk 4 above)
Comment (Mr-KayJayDee) [reply src/app/[locale]/(app)/_components/SharePopup.tsx:59]: @florian-lefebvre After reviewing this i didn't find any other way to do this, or maybe i did not understand your request 🤔

Comment (florian-lefebvre) [reply src/app/[locale]/(app)/_components/SharePopup.tsx:59]: You know what, that's not a big deal let's keep it like this

Comment (Mr-KayJayDee) [reply src/app/[locale]/(app)/_components/SharePopup.tsx:59]: Alright !

File: src/app/[locale]/onboarding/_components/Footer.tsx [hunk 5]
```diff
@@ -84,7 +84,7 @@ function TrustDialog({
                     className="w-full"
                     onClick={() => setOpen(false)}
                   >
-                    Got it!
+                    {t("gotIt")}
```
Comment (florian-lefebvre) [inline src/app/[locale]/onboarding/_components/Footer.tsx:87]: Let's move it to onboarding.shared.gotIt

File: src/app/[locale]/onboarding/loading/_components/QueueDisplay.tsx [hunk 6]
```diff
@@ -72,7 +72,7 @@ function QueueExplanationDialog({
                     className="w-full"
                     onClick={() => setOpen(false)}
                   >
-                    Got it!
+                    {t("gotIt")}
```
Comment (florian-lefebvre) [inline src/app/[locale]/onboarding/loading/_components/QueueDisplay.tsx:75]: Let's move it to onboarding.shared.gotIt

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]

**Review by florian-lefebvre — APPROVED** [review]

**Comment by Androz2091** [comment]
@Mr-KayJayDee can you solve conflicts?

**Comment by Mr-KayJayDee** [comment]
After this fix i might have to re create a PR with the latest changes on main. Because i need to re translate this part. If i do it now i guess i will have another conflict

**Review by florian-lefebvre — APPROVED** [review]
LGTM

**Comment by Mr-KayJayDee** [comment]
Don't merge it now ! I'll try something

**Review by florian-lefebvre — APPROVED** [review]
LGTM! @Androz2091 please review and merge if it looks good to you

**Review by Androz2091 — APPROVED** [review]
LGTM

**Pull request merged**